workflow/executor/docker/docker.go

```diff
@@ -86,6 +89,37 @@ func (d *DockerExecutor) GetOutputStream(containerID string, combinedOutput bool
 	return reader, nil
 }
 
+func (d *DockerExecutor) GetExitCode(containerID string) (int32, error) {
+	cmd := exec.Command("docker", "inspect", containerID, "--format='{{.State.ExitCode}}'")
+	log.Info(cmd.Args)
```

Comment:
I think this can be removed

Reply:
You mean logging the command args ?

Reply:
Either removing it or making it `Debug` and adding context to it would work. Maybe something like 
```suggestion
	log.Debug("Exit code docker command args", cmd.Args)
```